$H_0$: Travel time is the same or longer with the new bus line as it was before the new line.

$H_{\rm Alt}$: Travel time is shorter with the new bus line than it was before to a certainty of $p < 0.05$.

Based on prior information, average travel before the new bus line was 36 minutes with a standard deviation of 6. 100 measurements on the new line were taken. The standard deviation of the mean from that sample is the population standard deviation for individual measurements divided by the square root of the sample size. This would be $6/\sqrt{100}$ or $6/10$ or 0.6.

In [3]:
from __future__ import print_function

import numpy as np
import pandas as pd
import os

puidir = os.getenv("PUI2016")
hdir = puidir + "/HW3_ss4977"
infile = hdir + "/times.txt"
df = pd.read_csv(infile)
ser = df["Travel Time"]
stdpop = 6
sampstd = stdpop / 100**.5
ztest = (ser.mean() - 36) / sampstd
print (ztest)


-2.55639718617


The sample average travel time on the new bus route is 2.56 standard deviations below the average for the previous route. The $Z$-score with a $p$-value of 0.05 is just under 2, therefore, the null hypothesis is rejected with $p < 0.05$.
